From 26acea3b2cc8d9c62cc3a5801f145735240111a2 Mon Sep 17 00:00:00 2001
From: Zelan Zou <zelan.zou@nxp.com>
Date: Mon, 5 Jan 2026 15:37:37 +0900
Subject: [PATCH] Makefile.am: Add ignore file to EXTRA_DIST

Fix error:
| The following files are in git but not the distribution:
| .sourcegraph/ignore
| make[1]: *** [Makefile:6623: dist-hook-git] Error 1
| make[1]: *** Waiting for unfinished jobs....

Upstream-Status: Inappropriate
Signed-off-by: Zelan Zou <zelan.zou@nxp.com>
---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 08005289a..88e74d41a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -92,7 +92,8 @@ EXTRA_DIST = \
 	$(MAN_ROOTS) \
 	Vagrantfile \
 	Vagrantfile-FreeBSD \
-	.mailmap
+	.mailmap \
+	.sourcegraph/ignore
 bin_PROGRAMS =
 sbin_PROGRAMS =
 bin_SCRIPTS =
-- 
2.34.1

